CollAction/Frontend/app/project/ProjectList.tsx

```diff
@@ -21,7 +22,7 @@ class ProjectThumb extends React.Component<IProject, null> {
       backgroundImage: `url(${this.props.bannerImagePath})`,
     };
 
-    const link = `/Projects/Details/${this.props.projectId}`;
+    const link = `/projects/${encodeURIComponent(this.props.projectUriPart)}/${this.props.projectId}/details`;
```

Comment:
We should return the canonical url in the find API call, that way you can have consistent urls.

Reply:
Fixed